scripts/ci/check_compliance.py

```diff
@@ -251,7 +251,7 @@ class BoardYmlCheck(ComplianceTest):
     """
     name = "BoardYml"
     doc = "Check the board.yml file format"
-    path_hint = "<zephyr-base>"
+    path_hint = "<git-top>"
```

Comment:
This has no effect other than the reported directory? This check doesn't run for a module.

Reply:
It runs for modules when run in the module directory.

Reply:
Thanks for clarifying. I see what you mean now.

Reply:
Fixed!